bundles/org.openhab.binding.tsmart/README.md

```diff
@@ -0,0 +1,53 @@
+# Tesla T-Smart Binding
+
+This binding is designed to connect to the Tesla T-Smart Smart Immersion Heater / Thermostat device https://www.tsmart.co.uk/.
+
+The device allows monitoring and control over the temperature and heating of water in a water cylinder.
+
+The build is derived from the API docs shared here https://community.home-assistant.io/t/telsa-t-smart-immersion-heater/345853 and some inspiration from https://github.com/britkat1980/tsmart.
```

Comment:
You can remove this from the user documentation, if needed you can add it to the start post of the PR or create a developer/usefull links topic at the end of the docs.